x-pack/test/security_solution_cypress/cypress/tasks/api_calls/prebuilt_rules.ts

```diff
@@ -195,6 +198,18 @@ export const preventPrebuiltRulesPackageInstallation = () => {
   cy.intercept('POST', '/api/fleet/epm/packages/security_detection_engine/*', {});
 };
 
+export const addAvailableToInstallPrebuiltRules = (
```

Comment:
I think the name of the method is a little confusing, if you don't understand the whole idea of first installing the assets, and then installing those assets as prebuilt rules to Kibana.

What do you think about `makeRulesAvailableForInstall`?

Reply:
I ended up with `installPrebuiltRuleAssets()`. It better reflects the process under the hood and an explanation comment should give some info. Generally speaking we can change business logic later on so installed assets aren't available to be installed without a flag or something and the function names `makeRulesAvailableForInstall` won't do it its name says.